scalafmt-core/shared/src/main/scala/org/scalafmt/config/ScalafmtRunner.scala

```diff
@@ -89,6 +90,7 @@ object ScalafmtRunner {
         Scala211,
         scala212,
         scala213,
+        source3,
```

Comment:
should we just allow explicit `Scala213Source3` instead of the intermediate (and perhaps more ambiguous `source3`)?

Reply:
Sure, I can add both. I thought that shorter might be better, but maybe that's not the case here.

Reply:
I also added a small paragraph for it in the docs, what do you think?

Reply:
i think it's lovely!